chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -104,67 +109,84 @@ def setUp(self):
             self.remote_operations.enable_agent_debug(self.TEST_SERVERS)
 
         self.wait_until_true(self.supervisor_controlled_processes_running)
-        self.initial_supervisor_controlled_process_start_times = self.get_supervisor_controlled_process_start_times()
+        self.initial_supervisor_controlled_process_start_times = self.get_supervisor_controlled_process_start_times(
```

Comment:
really?

Reply:
What's the really for?

Reply:
I think it's the breaking up of the line by moving an empty close bracket to the next line.  Typo perhaps?

Reply:
just wasn't sure that putting an empty bracket on the new-line was intentional, makes it less readable

Reply:
All these changes are being done by https://github.com/google/yapf, which I'd argue we all should be using precisely to avoid bikesheding over preferred code-style. 

That said, it might be a bug in yapf as when I move the closing paren to the same line it doesn't reformat (only a complaint about line len).

Reply:
I've been using the pep linters built into pycharm so I will see if I can find a plug-in for that, definitely need to be using the same linter